example/PER_example.py

```diff
@@ -0,0 +1,142 @@
+
```

Comment:
This should be removed with PER_TD3 added into the network_factory.py to run through the existing example files. This doesn't add anything that the other example doesn't.